lib/pagy.rb

```diff
@@ -50,5 +50,6 @@ def series(size=@vars[:size])
 
 end
 
+require 'pagy/version'
```

Comment:
reuiring the version is not needed here, if you load directly the version in the gemspec